docs/getting-started/migrate/index.md

```diff
@@ -51,7 +52,32 @@ if [[ -f /HOME_DIR/.shopify-cli/shopify.sh ]]; then source /HOME_DIR/.shopify-cl
 [ -f "/HOME_DIR/.shopify-app-cli/shopify.sh" ] && source "/HOME_DIR/.shopify-app-cli/shopify.sh"
 ```
 
-Deleting or commenting out the relevant line in your shell profile will remove `shopify` as a command. You may need to reload your shell.
+Deleting or commenting out the relevant line in your shell profile will remove `shopify` as a command.
+
+### 3. Reload your terminal
```

Comment:
It seems like 3 comes before 2 here, or is it just the diff that looks confusing?

Reply:
Different levels (count the number of hashes!)